dt-metrics/combined/mapbox-maps.php

```diff
@@ -62,6 +59,7 @@ public function scripts() {
                 'translations' => [],
                 'settings' => [
                     'map_key' => DT_Mapbox_API::get_key(),
+                    'no_key_html_msg' => sprintf( __( 'To view this map, a mapbox key is needed; click <a href="%s">here</a> to add.', 'disciple_tools' ), site_url( '/wp-admin/admin.php?page=dt_mapping_module&tab=geocoding' ) ),
```

Comment:
@kodinkat we try to avoid passing html to the translator.
Instead we can pass the whole `<a href="link">__('here', 'dt')</a>` as the var.
And let's use `_x( 'to view...', 'to view ... click here to add', 'disciple_tools' )` so the translator can see what is happening.

Reply:
@kodinkat that way works too :D